.github/workflows/_test.yml

```diff
@@ -55,3 +55,32 @@ jobs:
           # grep will exit non-zero if the target message isn't found,
           # and `set -e` above will cause the step to fail.
           echo "$STATUS" | grep 'nothing to commit, working tree clean'
+
+      # If this PR is for a new, not-yet-published version,
+      # we want to run a build and prepare it for uploading to test PyPI.
+      - name: Check if version exists on PyPI
+        shell: bash
+        run: |
+          set -euxo pipefail
+
+          PKG_NAME="$(poetry version | cut -d' ' -f 1)"
```

Comment:
ubernit: should there be space after d before quote `-d ' '`

Reply:
Long time habit to never leave whitespace after the `-d` parameter to `cut` specifically. I'm not even sure why we always did that at Kensho... 😅 Thanks!